sdk-extension/opentelemetry-sdk-extension-aws/setup.cfg

```diff
@@ -42,6 +42,8 @@ install_requires =
     opentelemetry-api == 0.16.dev0
 
 [options.entry_points]
+install_requires =
+    requests ~= 2.24.0
```

Comment:
is this correct? this would be adding requests to entry points. I would have expected this be added to the section above.